src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/services/CaseUpdater.java

```diff
@@ -66,17 +77,22 @@ public SuccessfulUpdateResponse update(CaseUpdateRequest caseUpdateRequest) {
             originalCase.bulkScanCaseReference
         );
 
-        final List<String> warnings = updatedCaseValidator.getWarnings(newCase);
+        final List<String> resultWarnings =
+            newArrayList(Iterables.concat(warnings, updatedCaseValidator.getWarnings(newCase)));
```

Comment:
is `updatedCaseValidator.getWarnings(newCase))` still needed?

Reply:
not any more, you are right